src/db/effects_tables_vco.tsv

```diff
@@ -26,7 +26,7 @@ vco_nor_wul_marauder_spear_hunter_bonus_large	monster_hunt.png	101	monster_hunt.
 vco_nor_wul_marauder_spear_hunter_primal_instincts	dlc12_plaque_stage_3.png	101	dlc12_plaque_stage_3.png	campaign	true
 vco_nor_wul_movement_range_post_raze_port	engineer.png	101	engineer.png	campaign	true
 vco_nor_wul_units_unlock_dummy	chaos_gifts.png	101	chaos_gifts.png	campaign	true
-vco_nor_wul_ability_enable_youre_next	wh2_dlc15_lord_abilities_youre_next.png	1	wh2_dlc15_lord_abilities_youre_next.png	campaign	true
+vco_nor_wul_enable_rampage_ability	wh2_dlc15_lord_abilities_youre_next.png	1	wh2_dlc15_lord_abilities_youre_next.png	campaign	true
```

Comment:
Maybe naming it Hunter of Champions directly? This sounds like standard rampage.